src/transformer_util.ts

```diff
@@ -206,28 +207,31 @@ function emitMissingSyntheticCommentsAfterTypescriptTransform(context: ts.Transf
   };
 }
 
-function extractModuleNameFromRequireVariableStatement(node: ts.Node): string|null {
-  if (node.kind !== ts.SyntaxKind.VariableStatement) {
-    return null;
-  }
-  const varStmt = node as ts.VariableStatement;
-  const decls = varStmt.declarationList.declarations;
-  let init: ts.Expression|undefined;
-  if (decls.length !== 1 || !(init = decls[0].initializer) ||
-      init.kind !== ts.SyntaxKind.CallExpression) {
-    return null;
+function isCommonJsRequireStatement(node: ts.Node): boolean {
+  // CommonJS requires can be either "var x = require('...');" or (for side effect imports), just
+  // "require('...');".
```

Comment:
By the way, I think this code might be duplicated with the ES5 rewriter, which also must find require statements.  At least, it feels pretty familiar.

Reply:
https://github.com/angular/tsickle/blob/master/src/es5processor.ts#L247

Reply:
Yeah it's fine I think.  It's worth keeping in mind though (the other code I linked has a lot more stuff around export * that I think maybe doesn't apply here?).

Reply:
Ack, I actually added a test for that because I was worried. It turns out not to be a problem though, because we actually do rewrite those from `export *` to explicit exports for each symbol.